frontend/src/app/application-forms/application-noncommercial-group/noncommercial-mock.ts

```diff
@@ -0,0 +1,98 @@
+export const noncommercialMock = {
+  appControlNumber: 'a309fe16-47a4-4da7-a293-9697d438815f',
+  applicationId: 4,
+  createdAt: '2017-10-24T11:56:38.000Z',
+  applicantMessage: 'holding',
+  status: 'Hold',
+  authEmail: 'test@test.com',
+  authorizingOfficerName: '',
+  authorizingOfficerTitle: '',
+  revisions: [
+    {
+      revisionId: 11,
+      applicationId: 4,
+      applicationType: 'noncommercial',
+      status: 'Hold',
+      email: 'test@test.com',
+      createdAt: '2017-10-24T12:00:06.000Z',
+      updatedAt: '2017-10-24T12:00:06.000Z'
+    }
+  ],
+  district: '11',
+  region: '06',
+  forest: '05',
+  type: 'noncommercial',
+  eventName: 'Walk in the park',
+  signature: 'SA',
+  applicantInfo: {
+    addAdditionalPhone: true,
+    addSecondaryPermitHolder: true,
+    emailAddress: 'no@noemail.com',
+    organizationName: 'Test organization',
+    orgType: 'Corporation',
+    primaryAddressSameAsOrganization: false,
+    primaryFirstName: 'Micky',
+    primaryLastName: 'Watson',
+    secondaryAddressSameAsPrimary: false,
+    secondaryFirstName: 'Micky',
+    secondaryLastName: 'Watson',
+    website: 'http://test.com'
+    //   primaryAddress: {
```

Comment:
Could we please remove these commented sections.

Reply:
Removed.